docs/templates/getting-started/sequential-model-guide.md

```diff
@@ -137,6 +137,24 @@ model.compile(optimizer='rmsprop',
 # for a mean squared error regression problem
 model.compile(optimizer='rmsprop',
               loss='mse')
+
+# for custom metrics
+import keras.backend as K
+
+def mean_pred(y_true, y_pred):
+    return K.mean(y_pred)
+
+def precision_recall(y_true, y_pred):
+    precision = ...
+    recall = ...
+    return {
+        'precision': precision,
+        'recall': recall,
+    }
```

Comment:
Maybe mean incertitude of the predictions (distance to 0.5, if balanced binary classification problem). Maybe rate of false positives and false negatives. That last one can be pretty useful.